app/components/Img/LargeImage.js

```diff
@@ -4,7 +4,7 @@ const LargeImage = Img.extend`
   width: ${CELL * 5}px;
   height: ${CELL * 5}px;
 
-  @media only screen and (max-width: 768px) {
+  @media only screen and (max-width: 576px) {
```

Comment:
should be constant

Reply:
it is default bootstrap size grid for mobile, if it has sense it is needed to change everywhere